### PR TITLE
id:6468 comment: meta data for Debt Management Page

### PR DESCRIPTION
**Pull request by Guntrisoft**
- There was no translations provided, so for welsh:
  - Used the h1 title for the meta title on welsh,
  - Same english description as the welsh version

### CODE REVIEW

**Comment by neoeno** [comment]
Is there a reason we can't get this translated?

**Comment by Guntrisoft** [comment]
I believe its mainly to concentrate SEO optimisation for the page, and getting the welsh translation wasn't deemed worth the effort.

**Comment by neoeno** [comment]
Happy to get this translated for you if it's just that it's just been missed by the card-creator. I'll add it to my list if you like?

**Comment by Guntrisoft** [comment]
My completeness and correctness head says yes, and the other side of me says talk to Mani/Jacqui about it :)

File: config/locales/debt_management/debt_management.en.yml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 en:
+  meta_title: What to do if your debt management plan has stopped
```
Comment (asmega) [inline config/locales/debt_management/debt_management.en.yml]: i'm surprised this contents of this file is not namespaced eg...
```
en:
  campaigns:
    debt_management:
      translations: go here
```
as all the locale files will get munged together into one hash which I18n lib will use

I think we're lucky this isn't causing bugs or is causing bugs that we are not aware off...

although maybe outside the scope of this ticket this should be noted and fixed

**Comment by benbarnett** [comment]
:+1:

File: config/locales/debt_management/debt_management.en.yml (hunk 1 above)
Comment (benbarnett) [inline config/locales/debt_management/debt_management.en.yml]: nice, good spot

**Comment by tomas-stefano** [comment]
:+1:

**Pull request merged**